content/en/methods/filters.md

```diff
@@ -1263,7 +1263,7 @@ Replaces a filter's parameters in-place.
 ##### Headers
 
 Authorization
-: {{<required>}} Provide this header with `Bearer <user token>` to gain authorized access to this API method.
+: {{<required>}} Provide this header with `Bearer <user_token>` to gain authorized access to this API method.
```

Comment:
unrelated but i caught an earlier typo on line 1254, `or compatibility` should be `For compatibility`